integration/tests/consensus/sealing_test.go

```diff
@@ -235,11 +254,29 @@ SearchLoop:
 
 	receipt.ExecutorSignature = sig
 
-	// keep trying to send execution receipt to the first consensus node
+	// keep trying to send 2 matching execution receipt to the first consensus node
+	receipt2 := flow.ExecutionReceipt{
+		ExecutorID:        ss.exe2ID, // our fake execution node
+		ExecutionResult:   result,    // result for target block
+		Spocks:            nil,       // ignored
+		ExecutorSignature: crypto.Signature{},
+	}
+
+	id = receipt2.ID()
+	sig2, err := ss.exe2SK.Sign(id[:], exeUtils.NewExecutionReceiptHasher())
+	require.NoError(ss.T(), err)
+
+	receipt2.ExecutorSignature = sig2
+
+	valid, err := ss.exe2SK.PublicKey().Verify(receipt2.ExecutorSignature, id[:], exeUtils.NewExecutionReceiptHasher())
+	require.NoError(ss.T(), err)
+	require.True(ss.T(), valid)
+
 ReceiptLoop:
 	for time.Now().Before(deadline) {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 		err := ss.Execution().Send(ctx, engine.PushReceipts, &receipt, ss.conIDs...)
+		err = ss.Execution2().Send(ctx, engine.PushReceipts, &receipt2, ss.conIDs...)
```

Comment:
the consensus integration was failing because the sealing requires two matching receipts, but previously the integration tests only create one EN. 

Adding another EN solves the problem